updates to ECS docs per beta release (#11033)
hashicorp/consul: website/content/docs/ecs/get-started/requirements.mdx

# Requirements

The following requirements must be met in order to install Consul on ECS:

1. **Terraform:** The tasks that you want to add to the service mesh must first be modeled in Terraform.
1. **Launch Type:** Fargate and EC2 launch types are supported.

> 1. **Subnets:** ECS Tasks can run in private or public subnets. Tasks must have [network access](https://aws.amazon.com/premiumsupport/knowledge-center/ecs-pull-container-api-error-ecr/) to Amazon ECR to pull images.

Reviewer: ```suggestion
1. **Subnets:** ECS Tasks can run in private or public subnets. Tasks must have [network access](https://aws.amazon.com/premiumsupport/knowledge-center/ecs-pull-container-api-error-ecr/) to Amazon ECR or other public container registries to pull images.
```
These tasks can pull images from other container registries.

Reviewer 2: 👍 

Clarifying: This called out ECR in particular because that is where we publish our `consul-ecs` image - https://gallery.ecr.aws/hashicorp/consul-ecs for people to consume on AWS.

> 1. **Consul Servers:** You can use your own Consul servers or run servers inside ECS on Fargate using the `dev-server` Terraform module. The is a development/testing only server that does not support persistent storage.

Reviewer: ```suggestion
1. **Consul Servers:** You can use your own Consul servers (VM) or use HCP Consul. For development purposes or testing, you may use the the `dev-server` [Terraform module](https://github.com/hashicorp/terraform-aws-consul-ecs/tree/main). The is a development/testing only server that does not support persistent storage.
```
I think it's better if we make it clearer how Consul servers fit into this framework. Right now the only two non-dev options are Consul server running on VMs or HCP Consul.
